doc/faq.md

```diff
@@ -171,16 +175,16 @@ There're several ways:
 * `/etc/dpvs.conf`
 
 You can modify `dpvs.conf`, and refer `conf/xxx.sample`. Some parameters are configurable during
- run-time (on-fly), while others are configurable in initialization stage only. Refer to `conf/dpvs.conf.items` for all available parameters and corresponding type, default value and supported value range. Pls use `kill -SIGHUP <DPVS-PID>` to reload the former on-fly kind of parameters in `dpvs.conf`.
+ run-time (on-fly), while others are configurable in initialization stage only. Refer to [conf/dpvs.conf.items](../conf/dpvs.conf.items) for all available parameters and corresponding type, default value and supported value range. Pls use `kill -SIGHUP <DPVS-PID>` to reload the former on-fly kind of parameters in `dpvs.conf`.
 
 * `ipvsadm`, `keepalived`, `quagga`/`ospfd`/`bgpd`
 
-You should read `LVS`'s documents for `ipvsadm`/`keepalived` first, and note DPVS's version `ipvsadm`/`keepalived` are slightly different, pls check `tutorial.md` for details. For the configuration about `quagga`/`zebra`/`ospfd`/`bgpd`, pls refer `quagga`'s documents.
+You should read `LVS`'s documents for `ipvsadm`/`keepalived` first, and note DPVS's version `ipvsadm`/`keepalived` are slightly different, pls check [tutorial.md](./tutorial.md) for details. For the configuration about `quagga`/`zebra`/`ospfd`/`bgpd`, pls refer `quagga`'s documents.
```

Comment:
and note DPVS's version `ipvsadm`/`keepalived` **is** slightly different. **Please**